docs/snippets/angular/button-story-with-args.ts.mdx

```diff
@@ -2,7 +2,8 @@
 //Button.stories.ts
 
 const Template = (args: Button) => ({
-  props: args,
+  component: Button,
+  props: args
```

Comment:
```suggestion
  props: args,
```

Reply:
Fixed with the latest commit. Thank you for checking it :)